---
layout: page
title: Resume
---

[<i class="fa fa-file-alt" aria-hidden="true"></i> PDF](/alexbrohammer-resume.pdf)

## SKILLS

*Languages*: Python, R, Perl, SQL, Bash<br>
*Frameworks/tools*: Git, Tidyverse, pandas, Rstudio, Docker<br>
*Cloud services*: AWS (Lambda, S3, Batch, EC2, ECS, ECR, CodeCommit)<br>
*Bioinformatics*: Assembly (Canu, Hifiasm), Variant Calling (Samtools, GATK), Alignment (BWA, Minimap, Bowtie), Comparative Genomics (Mummer, MEME Suite, Blast, Orthofinder)<br> 

## EDUCATION

**B.S. Crop Sciences**<span style="float:right;">Aug 2009 – May 2012</span><br>
**M.S. Crop Sciences**<span style="float:right;">Aug 2012 – May 2014</span><br>
Department of Crop Sciences; University of Illinois, Urbana-Champaign<br>

**Ph.D. Applied Plant Sciences**<span style="float:right;">Aug 2014 – June 2018</span><br>
Department of Agronomy and Plant Genetics; University of Minnesota Twin Cities<br>

## EXPERIENCE

### Industry  

**Senior Data Scientist**<span style="float:right;">June 2018 – Present</span><br>
[Bayer Crop Science](https://www.cropscience.bayer.us/); Bayer; St. Louis, MO<br>  
**Corn Breeding Intern**<span style="float:right;">Dec 2013</span><br>  
Plant Breeding; BASF; Kauai, HI<br>  
**Plant Biotechnology Intern**<span style="float:right;">May 2011 – Aug 2011</span><br>  
Plant Biotechnology; Monsanto Comapany; St. Louis, MO<br>  
**Product Research and Development Intern**<span style="float:right;">May 2010 – Aug 2010</span><br>  
Plant Breeding; DuPont Pioneer; Ivesdale, IL<br>  

### Academia  

**Teaching Assistant**<span style="float:right;">2015</span><br>
Applied Plant Genomics and Bioinformatics; University of Minnesota Twin Cities<br>
**Teaching Assistant**<span style="float:right;">2014</span><br>
Global Food Production Web; University of Illinois, Urbana-Champaign<br>  

## PUBLICATIONS  

**2021**<br> 
Z Li, P Zhou, R Della Coletta, et al.<br>
*Single‐parent expression drives dynamic gene expression complementation in maize hybrids*<br>
[https://doi.org/10.1111/tpj.15042](https://doi.org/10.1111/tpj.15042)<br>

**2020**<br>
PJ Monnahan, J-M Michno, C O’Connor, et al.<br>
*Using multiple reference genomes to identify and resolve annotation inconsistencies*<br> 
[https://doi.org/10.1186/s12864-020-6696-8](https://doi.org/10.1186/s12864-020-6696-8)<br>

**2019**<br> 
A Singh, G Li, AB Brohammer, et al.<br> 
*Genome-wide association and gene co-expression network analyses reveal complex genetics of resistance to Goss’s wilt of maize*<br>
[https://doi.org/10.1534/g3.119.400347](https://doi.org/10.1534/g3.119.400347)<br>
SN Anderson, MC Stitzer, AB Brohammer, et al.<br>
*Transposable elements contribute to dynamic genome content in maize*<br>
[https://doi.org/10.1111/tpj.14489](https://doi.org/10.1111/tpj.14489)<br> 


**2018**<br>
AB Brohammer, TJY Kono, NM Springer et al.<br>
*The limited role of differential fractionation in genome content variation and function in maize (Zea mays L.) inbred lines*<br>
[https://doi.org/10.1111/tpj.13765](https://doi.org/10.1111/tpj.13765)<br>
J Bennetzen, S Flint-Garcia, CN Hirsch, et al.<br>
*The Maize Pan-Genome - The Maize Genome*<br>
[https://doi.org/10.1007/978-3-319-97427-9](https://doi.org/10.1007/978-3-319-97427-9)<br>
TJY Kono, AB Brohammer, SE McGaugh, et al.<br>
*Tandem duplicate genes in maize are abundant and date to two distinct periods of time*<br>
[https://doi.org/10.1534/g3.118.200580](https://doi.org/10.1534/g3.118.200580)<br>
CN Hirsch, CD Hirsch, AB Brohammer, et al.<br>

**2016**<br>
*Draft assembly of elite inbred line PH207 provides insights into genomic and transcriptome diversity in maize*<br>
[https://doi.org/10.1105/tpc.16.00353](https://doi.org/10.1105/tpc.16.00353)<br>
